src/plugins/application_config/server/index.ts

```diff
@@ -0,0 +1,11 @@
+import { PluginInitializerContext } from '../../../core/server';
```

Comment:
Looks like we are missing the license header here

Reply:
Added.